### ISSUE TITLE
Seems to not work for me?

### ISSUE THREAD

**Issue by charlike** [standard-things/esm#16]
Am i idiot and i'm not getting something or what? :D 

I have few (6-7 files) full esm, some of them import others of them and all of them `export default`. I have `"esm": "js"` option on, because don't want to rename them. Then i have one `test.js` file from which i am trying to get the result of `index.js` which `export default` a function.

```js
require('@std/esm')

console.log(require('./index.js')) // => { [Symbol(@std/esm:sourceType)]: 'module' }
console.log(require('./index.js').default) // => undefined
```

How to get the function that is exported by `index.js`?

![2017-08-11-13 56 26_1280x1024_scrot](https://user-images.githubusercontent.com/5038030/29210499-8ea1ab0c-7e9c-11e7-906f-6283d5a5eec5.png)
![2017-08-11-13 56 23_1280x1024_scrot](https://user-images.githubusercontent.com/5038030/29210500-8ebf2d4e-7e9c-11e7-8515-f0cf28a328c5.png)

**Comment by jdalton** [comment]
Hi @charlike!

It's not your fault. I believe this is related to (likely a dup of) #15.
I'm finishing up the patch so it should be fixed today :)

**Comment by charlike** [comment]
hm, strange, didn't get the notification.

Cool! I'll try it.

**Comment by jdalton** [comment]
Fixed with v0.3.0 🎉
